packages/vcardgen/vcardgen.1.0/opam

```diff
@@ -0,0 +1,12 @@
+opam-version: "1.2"
+maintainer: "lord@crocodile.org"
+authors: "Vadim Zaliva"
+homepage: "https://github.com/vzaliva/vcardgen"
+bug-reports: "https://github.com/vzaliva/vcardgen/issues"
+license: "BSD2"
+dev-repo: "https://github.com/vzaliva/vcardgen.git"
+build: ["jbuilder" "build"]
+install: ["jbuilder" "install"]
```

Comment:
This `install` field is not needed, as jbuilder will generate an `install` file for you.

Reply:
Thanks. I just removed it.